extensions/NV/SPV_NV_displacement_micromap.asciidoc

```diff
@@ -0,0 +1,297 @@
+SPV_NV_displacement_micromap
+============================
+
+Name Strings
+------------
+
+SPV_NV_displacement_micromap
+
+Contact
+-------
+
+See *Issues* list in the Khronos SPIRV-Registry repository:
+https://github.com/KhronosGroup/SPIRV-Registry
+
+Contributors
+------------
+
+- Eric Werness, NVIDIA
+- Ashwin Lele, NVIDIA
+- Pyarelal Knowles, NVIDIA
+- Christoph Kubisch, NVIDIA
+
+Status
+------
+
+- Provisional
```

Comment:
I don't have a super strong opinion about this, but we should think about any conventions and policies for "provisional" extensions.  For example, the `SPV_AMDX_shader_enqueue` extension includes `X` in its name and has a big disclaimer describing what is intended by "provisional".  Should we include similar text here?

Reply:
Sounds good, thanks! I saw the same blurb e.g. in the provisional [SPV_KHR_ray_query](https://raw.githubusercontent.com/KhronosGroup/SPIRV-Registry/798acbe508c32db0f4619cc1844e8762bab945a6/extensions/KHR/SPV_KHR_ray_query.asciidoc) (older version of the current file). We'd like to do the same and avoid an intermediate X version of the extension - I think that's what you're suggesting anyway.